test/api.test.ts

```diff
@@ -7,8 +7,8 @@ import * as nock from 'nock'
 describe('api', () => {
   test
   .nock('https://api.heroku.com', api => api
-    .get('/hello')
-    .reply(200, 'hello!')
+  .get('/hello')
+  .reply(200, 'hello!'),
```

Comment:
these commas are weird but it's what the eslint config provided by oclif desires 🤷